### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by jamie-lynch** [guardian/cdk#175] [Ideas]
Got some ideas for a new feature or spotted a gap where a construct or pattern should be? 💡 

Let us know and maybe it'll be coming to a release near you soon! 🆕 

To help show the status of the ideas, titles may contain the following icons:
🚢  - This component or feature has been implemented
✏️  - This component or feature is currently being worked on
📝 - This component or feature is on the backlog

**Comment by akash1810** [comment]
As more and more components bring their own parameters and read from "standard" locations in parameter store, should we have a tool that checks your AWS account is compatible? That is, checks your account has these parameter store values defined.

Without this, the experience isn't too bad as cloudformation will report an inability to lookup the parameter and the stack operation will fail.

**Reply by jamie-lynch** [reply]
That's a good idea. Maybe a good fit for https://github.com/guardian/cdk-cli?

I think you've suggested this already but probably a doc which lists all of the default locations would be good too.

**Reply by jamie-lynch** [reply]
https://github.com/guardian/cdk/pull/179 adds docs
